ccmain/thresholder.cpp

```diff
@@ -312,7 +312,7 @@ void ImageThresholder::ThresholdRectToPix(Pix* src_pix,
       bool white_result = true;
       for (int ch = 0; ch < num_channels; ++ch) {
         int pixel = GET_DATA_BYTE(const_cast<void*>(
```

Comment:
Ok, type casts for `GET_DATA_BYTE` can be removed after this PR has been applied.